pkg/report/report.go

```diff
@@ -303,18 +307,24 @@ func Generate(ctx context.Context, path string, mrs yara.MatchRules, ignoreTags
 	var caps []string
 	var syscalls []string
 
+	ignoreBincapz := false
 	overallRiskScore := 0
 	riskCounts := map[int]int{}
 	risk := 0
 	key := ""
 
 	for _, m := range mrs {
+		if m.Rule == BINARY && ignoreSelf {
+			ignoreBincapz = true
+		}
 		risk = behaviorRisk(m.Namespace, m.Rule, m.Tags)
 		if risk > overallRiskScore {
 			overallRiskScore = risk
 		}
 		riskCounts[risk]++
-		if risk < minScore {
+		// The bincapz rule is classified as harmless
+		// This will prevent the rule from being filtered
+		if risk < minScore && !ignoreBincapz {
```

Comment:
I spent way too long trying to figure out why we never got to the `__bincapz__` case statement. Since the `bincapz` rule is harmless and the default risk is `low`, the rule was being filtered.